src/jimp2/wireworld/z8/datamangment/Electron.java

```diff
@@ -27,7 +29,36 @@ public boolean validateSpace(World world) {
 
     @Override
     public void insertIntoWorld(World world) {
-        // TODO implement here
+        if(world.cells[position.x][position.y].getState()==State.CONDUCTOR) {
```

Comment:
ten if to do validate czy jak ta metoda się nazywała (możesz wkleić tam i zakomentować bo to na przyszłość) no i oczywiście trzeba tam też sprawdzić czy mieści się w ramach świata